covid19-etl/tests/test_jhu_to_s3.py

```diff
@@ -134,3 +152,113 @@ def test_jhu_to_s3():
         with open(os.path.join(uploaded_files_dir, "map_data", file)) as f:
             uploaded_data = json.loads(f.read())
         assert expected_data == uploaded_data
+
+
+def test_jhu_to_s3_illinois(clean_up):
+    # check which locations we have data for
+    all_illinois_counties = set()
+    for filename in [
+        "time_series_covid19_confirmed_global.csv",
+        "time_series_covid19_deaths_global.csv",
+        "time_series_covid19_recovered_global.csv",
+        "time_series_covid19_confirmed_US.csv",
+        "time_series_covid19_deaths_US.csv",
+    ]:
+        with open(os.path.join(INPUT_DATA_DIR, filename), "r") as f:
+            reader = csv.DictReader(f, delimiter=",", quotechar='"')
+            for row in reader:
+                country = None
+                state = None
+                for key in ["Country/Region", "Country_Region"]:
```

Comment:
Can we use a ternary if condition instead of a `for`, `if`, `break` since there are only two fields to iterate and there are no other operations other than assignment.
Something like
```
country = row['Country/Region'] if 'Country/Region' in row else row['Country_Region']
```
same thing for `state`

Reply:
my version also handles the cases of `row` not containing either key, and of `row[key]` being null, but that shouldn't happen here